prow/config.yaml

```diff
@@ -316,6 +316,7 @@ tide:
     kyma-incubator/sc-removal: squash
     kyma-incubator/hydra-login-consent: squash
     kyma-incubator/website-vuepress: squash
+    kyma-incubator/migrate: squash
```

Comment:
Edit this in `templates/templates/prow-config.yaml` and run ` make jobs-definitions`

Reply:
Thanks 👍